database/database_test.go

```diff
@@ -119,3 +123,39 @@ func testCreateBlockedSkylink(t *testing.T) {
 		t.Fatal("not equal")
 	}
 }
+
+// testIsAllowListedSkylink tests the 'IsAllowListed' method on the database.
+func testIsAllowListedSkylink(t *testing.T) {
```

Comment:
F/U: test with  base32 skylink

Reply:
not necessary we add base64 V1s manually